src/components/IncidentDetails/ResponderTypes/Jira/index.tsx

```diff
@@ -1,58 +1,115 @@
+          type="Jira"
           control={control}
-          name="project"
-          rules={{
-            required: "Please provide valid value"
+          name="configType"
+          autoFetch={true}
+          onSelect={(e: any) => {
+            setJiraProjectType(e);
+            setValue("project", "");
+            setValue("issueType", "");
           }}
-          render={({ field }) => {
-            const { onChange, value } = field;
-            return (
-              <TextInput
-                label="Project"
-                id="project"
-                className="w-full"
-                onChange={onChange}
-                value={value}
-              />
-            );
-          }}
-        />
-        <p className="text-red-600 text-sm">{errors.project?.message}</p>
-      </div>
-      <div className="mb-4">
-        <Controller
-          control={control}
-          name="issueType"
+          label={
+            <label className="block text-sm font-bold text-gray-700 mb-2">
+              Jira Config Type
+            </label>
+          }
+          value={jiraProjectType}
+          id="config-type"
           rules={{
             required: "Please provide valid value"
           }}
-          render={({ field }) => {
-            const { onChange, value } = field;
-            return (
-              <TextInput
-                label="Issue Type"
-                id="issueType"
-                className="w-full"
-                onChange={onChange}
-                value={value}
-              />
-            );
-          }}
-        />
-        <p className="text-red-600 text-sm">{errors.issueType?.message}</p>
+        >
+          <p className="text-red-600 text-sm">{errors?.configType?.message}</p>
+          <ConfigItem
+            type="Jira"
+            control={control}
+            name="project"
+            value={jiraProject}
+            autoFetch={false}
+            onSelect={(selected) => {
+              setJiraProject(selected);
+              setValue("issueType", "");
+            }}
+            itemsPath="$..projects[*]"
+            namePath="$.name"
+            valuePath="$.name"
+            label={
+              <label
+                htmlFor="project"
+                className="block text-sm font-bold text-gray-700 mb-2 mt-4"
+              >
+                Project
+              </label>
+            }
+            isDisabled={!jiraProjectType}
+            id="project"
+            rules={{
+              required: "Please provide valid value"
+            }}
+          >
+            <p className="text-red-600 text-sm">{errors?.project?.message}</p>
+            <ConfigItem
+              type="Jira"
+              control={control}
+              name="issueType"
+              value={issueType}
+              autoFetch={false}
+              onSelect={(selected) => {
+                setIssueType(selected);
+              }}
+              itemsPath="$..task.status[*]"
```

Comment:
This is wrong - it selects all status from all projects, it needs to show only the issue types for the selected project,